core/src/test/java/org/mskcc/oncotree/utils/MainTypesUtilTest.java

```diff
@@ -35,7 +35,7 @@
  */
 
 @RunWith(SpringRunner.class)
-@Import(OncotreeUtilTestConfig.class)
+@Import(OncotreeTestConfig.class)
```

Comment:
Good .. I had thought to rename the shared config to something more general. Perfect